README.md

```diff
@@ -30,19 +31,33 @@ query and filter sample data via REST.
 
 #### Application information
 
-- Name: `loopback-example-model-relations`
-- Directory to contain the project: `loopback-example-model-relations`
-
 ```
 $ slc loopback loopback-example-model-relations
 ... # follow the prompts
+- Name: `loopback-example-model-relations`
+- Directory to contain the project: `loopback-example-model-relations`
 $ cd loopback-example-model-relations
 ```
+### Create the datasource
+
+```
+$ slc loopback:datasource
+... # follow the prompts, choose `other` to define custom connctor
+```
+- Name: `transient`
+- Connector: `other`
+  - Name: `transient`
```

Comment:
If you're going to keep this style, then I put the list *before* the commands (so they know what to type in when they get to "# follow the prompts").